docs/reference/configuration.md

```diff
@@ -162,7 +162,8 @@ This reference uses `.` to denote the nesting of values.
 * `graveler.commit_cache.size` `(int : 50000)` - How many items to store in the commit cache.
 * `graveler.commit_cache.ttl` `(time duration : "10m")` - How long to store an item in the commit cache.
 * `graveler.commit_cache.jitter` `(time duration : "2s")` - A random amount of time between 0 and this value is added to each item's TTL.
-* `graveler.background.rate_limit` `(int : 0)` - Advence configuration to control background work done rate limit in requests per second (default: 0 - unlimited).
+* `graveler.max_batch_delay` `(duration : 3ms)` - Controls the server batching period for references store operations.
+* `graveler.background.rate_limit` `(int : 0)` - Advance configuration to control background work done rate limit in requests per second (default: 0 - unlimited).
```

Comment:
- "Advanced" not "Advance", I think.
- Actually I don't understand why we state _any_ configuration is "advanced", let alone why this one is.

```suggestion
* `graveler.background.rate_limit` `(int : 0)` - Requests per seconds limit on background work performed (default: 0 - unlimited).
```

Ideally we would also document _what_ is being done in the background - but certainly not in this PR.